.changeset/shiny-drinks-impress.md

```diff
@@ -0,0 +1,3 @@
+---
+---
+ci: add PR title length validation
```

Comment:
FYI you don't have to add changesets for PRs that don't change any of the packages.

Reply:
Noted.